src/api-binder.ts

```diff
@@ -912,7 +920,15 @@ export class APIBinder {
 		router.post('/v1/update', (req, res) => {
 			apiBinder.eventTracker.track('Update notification');
 			if (apiBinder.readyForUpdates) {
-				apiBinder.getAndSetTargetState(req.body.force, true).catch(_.noop);
+				this.config.get('instantUpdates').then(instantUpdates => {
+					if (instantUpdates) {
+						apiBinder.getAndSetTargetState(req.body.force, true).catch(_.noop);
+					} else {
+						console.log(
+							'Ignoring update notification because instant updates are disabled',
```

Comment:
Perhaps we should return a different http header, for example `405 - method not allowed` ?

Reply:
@CameronDiver maybe a different code would make sense, but I think that can cause issues since the balena API won't be expecting that?

Reply:
I guess we could use a 202 Accepted?

> The request has been accepted for processing, but the processing has not been completed. The request might or might not eventually be acted upon, as it might be disallowed when processing actually takes place.

From https://www.restapitutorial.com/httpstatuscodes.html

Cause I wouldn't want to consider this case an error.

Reply:
Yeah I think that makes sense, I would just like a method of detecting that this isn't going to work. If the API doesn't support it then that needs to be changed IMO.